scripts/superdesk/editor2/add-content.js

```diff
@@ -19,11 +19,15 @@ function($window) {
             // initialize state
             vm.updateState();
             // listen for update state signals
-            scope.$parent.$on('sdAddContent::updateState', function(signal, event, editorElem) {
+            var listener = scope.$parent.$on('sdAddContent::updateState', function(signal, event, editorElem) {
                 vm.updateState(event, editorElem);
             });
             // update on resize
             angular.element($window).on('resize', vm.updateState);
+            scope.$on('$destroy', function() {
+                angular.element($window).off('resize', vm.updateState);
+                listener();
```

Comment:
i think it should be stated a bit more explicitly what it's a de-registration function, mb rename it to `unbindListener`